aplikigo-1/tasks/phase-1/3-gir-upgrade/INSTRUCTIONS.md

```diff
@@ -66,3 +66,84 @@ The following instructions should only be executed after the upgrade time
     ```
     sudo service regen start
     ```
+
+# Cosmovisor setup for Gir upgrade
```

Comment:
Let's make this as a RECOMMENDED option and add a NOTE to follow this before the upgrade time (after the proposal goes through)

```suggestion
# Option-1: Cosmovisor setup for Gir upgrade
```